core/shared/utils/uncommon/bh_getopt.c

```diff
@@ -0,0 +1,51 @@
+/* *****************************************************************
+*
+* Copyright 2016 Microsoft
+*
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This file's copyright is Microsoft, and the license is Apache 2.0, but not Apache-2.0 WITH LLVM-exception, it seems that it is not very difficult to implement getopt function, could you please rewrite the API?

Reply:
I just found this piece of code had an issue that optind will not increment if no ':' followed. We will reimplement it and fix the issue.